pkg/api/team_members.go

```diff
@@ -130,16 +135,14 @@ func (hs *HTTPServer) RemoveTeamMember(c *models.ReqContext) response.Response {
 		return response.Error(http.StatusBadRequest, "userId is invalid", err)
 	}
 
-	if err := hs.teamGuardian.CanAdmin(c.Req.Context(), orgId, teamId, c.SignedInUser); err != nil {
-		return response.Error(403, "Not allowed to remove team member", err)
-	}
-
-	protectLastAdmin := false
-	if c.OrgRole != models.ROLE_ADMIN {
```

Comment:
The check for whether the last admin isn't being removed is now done directly from [team SQL store](https://github.com/grafana/grafana/pull/43991/files#diff-9ff4b79076656e607d76f40213e6a3cff3e4bbf1ec0deda50dee277b25ebaae3R394).